### ISSUE TITLE
volgroup_required option

### ISSUE THREAD

**Issue by seraf1** [quattor/template-library-core#73]
Since the 14.8.0 version of the fichie rquattor / aii / ks / config ends with the line:
"/system/aii/osinstall/ks/volgroup_required" = true;

Ks files generated and always have the volgroup inserted rows but sl6 and 7, which causes an error in the installation.

If I change this line as follows:
"/system/aii/osinstall/ks/volgroup_required" ?= true;

Then the files to sl6 ks and 7 no longer have these insertions, which is okay.

Regards.

**Comment by jouvin** [comment]
I tend to agree that this line should be a default, not an enforced value. And it clearly makes the value define in the variant for el6 and el6 useless (`false`). I am just wondering why nobody reported this problem before... Could another KS expert comment?

**Comment by seraf1** [comment]
Why has this been removed?

**Comment by jrha** [comment]
From the 16.2 milestone? Because there was no PR to fix it, we made a decision some time ago to only set milestones for issues that are being worked on and leave the rest as a proper backlog.

**Comment by seraf1** [comment]
It is for all versions.

I have to make changes in the code each time I make an update of Quattor

**Comment by jrha** [comment]
Open a PR and it will be merged.
